services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java

```diff
@@ -270,7 +271,21 @@
     @GetMapping(path = "/{switch_id}/rules/synchronize")
     @ResponseStatus(HttpStatus.OK)
     public CompletableFuture<RulesSyncResult> syncRules(@PathVariable(name = "switch_id") SwitchId switchId) {
-        return switchService.syncRules(switchId);
+        return switchService.syncRules(switchId, false);
+    }
+
+    /**
+     * Synchronize (install) missing flows that should be on the switch but exist only in neo4j.
+     * Optionally removes excess rules from the switch.
+     *
+     * @return the synchronization result.
+     */
+    @ApiOperation(value = "Synchronize rules on the switch", response = RulesSyncResult.class)
+    @PostMapping(path = "/{switch_id}/synchronize")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<RulesSyncResult> syncRules(@PathVariable(name = "switch_id") SwitchId switchId,
```

Comment:
Why we return RulesSyncResult? It should return sth similar to `SwitchValidationResult` but about sync

Reply:
fixed